discord_bots/commands.py

```diff
@@ -2540,6 +2502,99 @@ async def mockqueue(ctx: Context, queue_name: str, count: int):
+    )
+    if not in_progress_game:
+        await send_message(
+            message.channel,
+            embed_description=f"Could not find game: {game_id}",
+            colour=Colour.red()
+        )
+        return
+    
+    team0_ipg_players: list[InProgressGamePlayer] = session.query(
+        InProgressGamePlayer
+    ).filter(
+        InProgressGamePlayer.in_progress_game_id == in_progress_game.id,
+        InProgressGamePlayer.team == 0,
+    )
+    team1_ipg_players: list[InProgressGamePlayer] = session.query(
+        InProgressGamePlayer
+    ).filter(
+        InProgressGamePlayer.in_progress_game_id == in_progress_game.id,
+        InProgressGamePlayer.team == 1,
+    )
+    team0_player_ids = set(map(lambda x: x.player_id, team0_ipg_players))
+    team1_player_ids = set(map(lambda x: x.player_id, team1_ipg_players))
+    team0_players: list[Player] = session.query(Player).filter(Player.id.in_(team0_player_ids))  # type: ignore
+    team1_players: list[Player] = session.query(Player).filter(Player.id.in_(team1_player_ids))  # type: ignore
+
+    in_progress_game_channels: list[InProgressGameChannel] = session.query(
+        InProgressGameChannel
+    ).filter(
+        InProgressGameChannel.in_progress_game_id == in_progress_game.id
+    )
+
+    for player in team0_players:
+        if player.move_enabled:
+            member: Member | None = guild.get_member(player.id)
+            if member:
+                channel: VoiceChannel | None = guild.get_channel(in_progress_game_channels[0].channel_id)
+                if channel:
+                    try:
+                        await member.move_to(channel, reason = game_id)
+                    except Exception as e:
+                        print(f"Caught exception sending message: {e}")
+
+    for player in team1_players:
+        if player.move_enabled:
+            member: Member | None = guild.get_member(player.id)
+            if member:
+                channel: VoiceChannel | None = guild.get_channel(in_progress_game_channels[1].channel_id)
+                if channel:
+                    try:
+                        await member.move_to(channel, reason = game_id)
+                    except Exception as e:
+                        print(f"Caught exception sending message: {e}")
+
+
+@bot.command(usage="<game_id>")
+@commands.check(is_admin)
+async def movegameplayers (ctx: Context, game_id: str):
+    """
+    Move players in a given in-progress game to the correct voice channels
+    """
+    message = ctx.message
+
+    if not config.ENABLE_VOICE_MOVE:
+        await send_message(
+            message.channel,
+            embed_description="Voice movement is disabled",
+            colour=Colour.red()
+        )
+        return
+    else:
+        await _movegameplayers(game_id, ctx)
+        await send_message(
+        message.channel,
+        embed_description=f"Players moved to voice channels for game {game_id}",
+        colour=Colour.green()
+    )
```

Comment:
can you see if you have the commit hook setup? look up `darker` in the readme

Reply:
Will get this setup for the future 👍